src/test/java/uk/gov/companieshouse/company_appointments/service/CompanyAppointmentServiceTest.java

```diff
@@ -435,7 +437,7 @@ void testFetchAppointmentsForCompanyThrowsBadRequestExceptionIfRegisterViewAndIn
     }
 
     @Test
-    void testFetchAppointmentsForCompanyThrowsNotFoundExceptionIfNoCountsInMetrics()
+    void testFetchAppointmentsForCompanyReturnsBaseResponseIfNoCountsOrNullCountsInMetrics()
```

Comment:
Ass above, 'BaseDocument' won't mean to much to a furure developer, so the name should be clearer

Reply:
addressed in most new commit